person.rb

```diff
@@ -7,15 +7,14 @@ def initialize(age, name = 'unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
-  attr_reader :id
+  attr_reader :id, :age, :rentals
   attr_accessor :name
 
```

Comment:
- [ ] As requested by the previous reviewer, Please make sure the `name` variable is writable also to work for the associations as expected.

Reply:
well resolved! ✔️